lib/cisco_node_utils/cisco_cmn_utils.rb

```diff
@@ -107,44 +122,20 @@ def self.process_network_mask(network)
       network
     end
 
-    def self.depth(a)
-      return 0 unless a.is_a?(Array)
-      1 + depth(a[0])
-    end
-
     # Helper to build a hash of add/remove commands for a nested array.
     # Useful for network, redistribute, etc.
     #   should: an array of expected cmds (manifest/recipe)
     #  current: an array of existing cmds on the device
     def self.delta_add_remove(should, current=[])
```

Comment:
@RichWellum Can you comment on this? Is this correct or is it reverting the code you added?